.eslintrc.js

```diff
@@ -328,6 +328,14 @@ module.exports = {
         'examples/vue/**/*.js',
         'examples/vue3/**/*.js',
       ],
+      rules: {
+        'no-unused-vars': [
+          'error',
+          {
+            'varsIgnorePattern': 'React',
+          },
+        ],
+      },
```

Comment:
within the react-native-expo example "React" is considered an unused var, but if it's removed I get the following error. 
The fix should actually be smarter and not just a rule exclusion

![Simulator Screen Shot - iPhone 12 Pro Max - 2022-11-01 at 16 50 52](https://user-images.githubusercontent.com/9303791/199278595-24b6c4a4-4377-41fb-b165-34b5d3d59557.png)